src/pages/dashboard/daily-customers/NewCustomers.tsx

```diff
@@ -1,30 +1,17 @@
 import { theme } from 'antd';
-import {
-  BarElement,
-  CategoryScale,
-  Chart,
-  ChartOptions,
-  ChartType,
-  Legend,
-  LinearScale,
-  Scale,
-  ScriptableContext,
-  Title,
-  Tooltip,
-} from 'chart.js';
+import { ChartOptions, ChartType, Scale, ScriptableContext } from 'chart.js';
+import { memo } from 'react';
 import { Bar } from 'react-chartjs-2';
-import { ChartUnit, DateFilter } from 'stores';
-import { createGradientChart, getLabelChart } from 'utils/chart';
+import { ChartUnit, FilterDateParams } from 'stores';
+import { createGradientChart, getLabelChart } from 'utils';
 
 type Props = {
   data: ChartUnit[];
   height: number;
-  filter?: DateFilter;
+  filter: FilterDateParams;
 };
 
-Chart.register(CategoryScale, LinearScale, BarElement, Title, Tooltip, Legend);
-
-export const DailyCustomerChart = ({ data, height, filter }: Props) => {
+const DailyCustomerChartMemo = ({ data, height, filter }: Props) => {
```

Comment:
I think I need to write some code **const DailyCustomerChart = memo(() => ...)**